### ISSUE TITLE
RADIUS log : create sections for search fields or order alphabetically

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#1136]
A bit hard to follow right now and to find what you are looking for.

Easiest would be to sort them but dividing them in sections would be nice

**Comment by jrouzierinverse** [comment]
Can please explain a bit more.
Because I do not know what you want to sort or divide.

**Comment by jrouzierinverse** [comment]
Never mind I did not read the title

**Comment by jrouzierinverse** [comment]
Fixed in cbd10207331e3f152f628637f6c9a9e8f9c45dfa
